tsdb/shard.go

```diff
@@ -591,10 +591,7 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]models.Point,
 	var droppedKeys [][]byte
 	if err := engine.CreateSeriesListIfNotExists(keys, names, tagsSlice); err != nil {
 		switch err := err.(type) {
-		// TODO(jmw): why is this a *PartialWriteError when everything else is not a pointer?
-		// Maybe we can just change it to be consistent if we change it also in all
-		// the places that construct it.
-		case *PartialWriteError:
+		case PartialWriteError:
```

Comment:
Is there a unit test somewhere that can cover this?

Reply:
Seems not for the error type. I will add a test case in index/inmem since this error is only populated from inmem index